pytheus/metrics.py

```diff
@@ -54,6 +53,10 @@ def __init__(
 
         # this will register to the collector
 
+    @property
```

Comment:
I would add the TODO: above this line explaining what this is and what it could be in the future

for the name maybe we could call it `type_` ? I'm always hesitant when naming something like a builtin but it might not be mandatory